WordPress/gutenberg
# Query Pagination: Add border and spacing support

## What?
Add border block support to the Pagination block.
Part of https://github.com/WordPress/gutenberg/issues/43247
Part of https://github.com/WordPress/gutenberg/issues/43243

## Why?
The pagination block is missing border support.

## How?
Adds the border block support in block.json

## Testing Instructions

1. Go to the Global Styles setting ( under appearance > editor > styles > edit styles > blocks )
2. Make sure that the Pagination block's border is configurable via Global Styles
3. Verify that Global Styles are applied correctly in the editor and frontend
4. Edit the template/page, Add a pagination block and Apply the border styles
5. Verify that block styles take precedence over global styles
6. Verify that block borders display correctly in both the editor and frontend
